src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,115 @@
+package Duke;
+
+import Task.Task;
+import Task.ToDo;
+import Task.Deadline;
+import Task.Event;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.io.FileWriter;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.ArrayList;
+
+public class Storage {
+
+    protected Path filePath;
+
+    /**
+     * Default constructor for Storage object.
+     */
+    public Storage() {
+        try {
+            Path dirToCreate = Paths.get("").resolve("output");
+            Path subDir = Files.createDirectories(dirToCreate); // create subdirectory for output list
+            this.filePath = Paths.get(subDir + "/" + "list.txt");
+            boolean fileExists = Files.exists(this.filePath);
+            if (!fileExists) { // if file does not exist, create the file
+                Files.createFile(this.filePath);
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+        }
+    }
+
+    /**
+     * Loads list of tasks saved from previous execution of program.
+     *
+     * @return a list of Tasks loaded from previous runs
+     */
+    public ArrayList<Task> load() {
+        ArrayList<Task> list = new ArrayList<>(); // init arraylist
+
+        try {
+            FileReader fr = new FileReader(this.filePath.toString());
+            BufferedReader br = new BufferedReader(fr);
+            StringBuffer sb = new StringBuffer();
+            String currLine;
+
+            while ((currLine = br.readLine()) != null) {
```

Comment:
This code block is a little difficult to read. Maybe include some line breaks between each condition?